diff.sh

```diff
@@ -10,5 +10,6 @@ do
     if $output
     then
         nbdiff master $f --output `$f.diff`
```

Comment:
If you were trying to output the diff to a file `filename.ipynb.diff`, you could try this:
```suggestion
        nbdiff master $f --out "$f.diff"
```